examples/properties/struct.cs

```diff
@@ -0,0 +1,4 @@
+public class DTO
```

Comment:
This seems like it tests something that is already tested.

Reply:
Should be `public struct DTO`, corrected.